code/modules/vehicles/wheelchair.dm

```diff
@@ -0,0 +1,118 @@
+/obj/vehicle/ridden/wheelchair
+	name = "wheelchair"
+	desc = "A chair with big wheels. It looks like you can move in this on your own."
+	icon = 'icons/obj/vehicles.dmi'
+	icon_state = "wheelchair"
+	layer = OBJ_LAYER
+	max_integrity = 100
+	armor = list("melee" = 10, "bullet" = 10, "laser" = 10, "energy" = 0, "bomb" = 10, "bio" = 0, "rad" = 0, "fire" = 20, "acid" = 30)
+	legs_required = 0
+	arms_required = 0
+	canmove = TRUE
+	density = FALSE
+	var/icon_overlay = "wheelchair_overlay"
+	var/list/drive_sounds = list('sound/effects/roll.ogg')
+	var/mob/living/carbon/human/H
+	var/mob/living/user
+	movedelay = 8
```

Comment:
This var doesn't even get used since you ditched Hippie's driver_move proc in favor of trying to make your code look original and brand new